msg/ExternalCollisionProtectionNameList.msg

```diff
@@ -0,0 +1,9 @@
+# A message for setExternalCollisionProtectionEnabled in External-Collision avoidance API
+
+uint8 number
```

Comment:
Besides I am confused with the message name (NameList, but the message is not a List and does not contain any name)